test/TestHistogram.cc

```diff
@@ -15,9 +15,10 @@
 #include "Timer/Timer.h"
 
 using namespace std;
-random_device rd;
-mt19937 mt(rd());
-uniform_real_distribution<float> float_random(0, 1.0f);
+
+random_device hist_rd;
+mt19937 hist_mt(hist_rd());
+uniform_real_distribution<float> hist_float_random(0, 1.0f);
```

Comment:
I don't get the purpose of this change, just naming?

Reply:
Yes, just because there are now multiple random generators in the different source files of the newly combined test target. Although I suspect we should make better use of test fixtures for this.

Reply:
Ok, I get it. Still, I liked letting my development environment generate just the test of interest with `cmake -Dtest=ON -DtestTileEncoding=OFF -DtestTable=OFF -DtestTimer=OFF =DtestBlockSmooth=OFF -DtestHistogram=ON ..`

Reply:
I think the friction of having a separate target for each test suite was part of the reason tests weren't getting written. For now, I guess you can just use the command-line args to run a specific test.

Reply:
I've added test fixtures, so the names are back to how they were before.